infra/modules/services/alerts.bicep

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// This template is used to create an alert.
+targetScope = 'resourceGroup'
+
+// Parameters
+param adfPipelineFailedAlertName string
+param datafactoryScope string
+param location string
+param tags object
+param processingService string
+param synapsePipelineFailedAlertName string
+param synapseScope string
+param dataFactoryEmailActionGroup string
+param dataProductTeamEmail string
+param cosmosRequestLimitedAlertName string
+param cosmosDBScope string
+param enableCosmos bool
+
+// Variables
+
+// Resources
+resource actiongroup 'Microsoft.Insights/actionGroups@2021-09-01' = {
+  name: dataFactoryEmailActionGroup
+  location: 'global'
+  tags: tags
+  properties: {
+    groupShortName: 'emailgroup'
+    emailReceivers: [
+      {
+        emailAddress: dataProductTeamEmail
+        name: 'emailaction'
+        useCommonAlertSchema: true
+      }
+    ]
+    enabled: true
+  }
+}
```

Comment:
What happens if no email was provided here?

Reply:
Alert module has a check for email